descwl_shear_sims/sim.py

```diff
@@ -221,6 +233,10 @@ def make_exp(
     star_bleeds=False,
     sky_n_sigma=None,
     draw_method='auto',
+    galsim_rng=None,
```

Comment:
We do not need this.

Reply:
You right, I forgot to remove the keyword. It should not be used in the code.